packages/tools/webpack-fluid-loader/src/getUrlResolver.ts

```diff
@@ -68,6 +72,7 @@ export function getUrlResolver(
 		case "tinylicious": {
 			const urls = tinyliciousUrls(options);
 			return new InsecureUrlResolver(
+				"",
```

Comment:
I imagine this might work... the concept of the deltaStreamUrl already exists, right? It's just never been used in tinylicious, probably because it wasn't necessary? I think it might be better to update the return object from `tinyliciousUrls()` to also have it set explicitly, even if it's the same URL as the other three.